Library/src/Web/MaintenanceApi/MaintenanceApi.cpp

```diff
@@ -39,9 +39,20 @@ void MaintenanceApi::Query(csp::common::String CHSEnvironment,
 	// S3 bucket URLS are case Sensitive so the CHS Environment name is set to lower case.
 	std::string CHSEnvironmentLower = std::string(CHSEnvironment.c_str());
 	std::transform(CHSEnvironmentLower.begin(), CHSEnvironmentLower.end(), CHSEnvironmentLower.begin(), ::tolower);
+	csp::web::Uri Uri;
+
+	if (CHSEnvironmentLower != "oprod")
```

Comment:
Can we add a comment explaining that for magnopus cloud hosted services, maintenance windows come from a different URL in prod, and so we handle this special case here?

Reply:
Should be good to go.

Reply:
Perfect, thanks